src/scripts/gamemodes/CaptureTheFlag/FlagStation.gd

```diff
@@ -0,0 +1,44 @@
+extends Area2D
+
+
+onready var Flag = preload("res://src/components/gamemodes/CaptureTheFlag/Flag.tscn")
+export var team_id = 0
+export var has_flag = true
+
+
+func _ready():
+# warning-ignore:return_value_discarded
+	GameEvent.connect("CTF_return_flag", self, "_on_CTF_return_flag")
+# warning-ignore:return_value_discarded	
+	GameEvent.connect("CTF_capture_flag", self, "_on_CTF_capture_flag")
+	if has_flag:
+		spawn_flag()
+
+
+func setup(_team_id):
```

Comment:
setup() runs before _ready() so maybe you should move setup() up, unless it's a convention thing then tell me.

Reply:
Builtin functions should be on top according to the official gdscript [guidelines](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_styleguide.html)